app/javascript/packs/public.jsx

```diff
@@ -325,6 +325,26 @@ delegate(document, '.sidebar__toggle__icon', 'keydown', e => {
   }
 });
 
+const setInputDisabled = (input, disabled) => {
+  input.disabled = disabled;
+
+  const wrapper = input.closest('.with_label');
+  if (wrapper) {
+    wrapper.classList.toggle('disabled', input.disabled);
+
+    const hidden = input.type === 'checkbox' && wrapper.querySelector('input[type=hidden][value="0"]');
+    if (hidden) {
+      hidden.disabled = input.disabled;
+    }
+  }
+};
+
+delegate(document, '#account_statuses_cleanup_policy_enabled', 'change', ({ target }) => {
+  [].forEach.call(target.form.querySelectorAll('input:not([type=hidden],#account_statuses_cleanup_policy_enabled), select'), (input) => {
```

Comment:
You should be able to write `target.form.querySelectorAll(…).forEach()` here, this is easier to read

Reply:
This pattern was introduced in #3179. I can remove it (possibly a separate PR), if we are ready to abandon support for [Edge 15](https://caniuse.com/mdn-api_nodelist_foreach).

Do we have a list of officially supported browsers? I think it would be useful, so we don't spend a lot of effort on supporting browser X in one part of the code but has abandoned it elsewhere. Also, testers will know what is relevant to test.

Reply:
According to the `.browserslist` configuration file, we support: [`defaults not IE 11 not dead`](https://browsersl.ist/#q=defaults%0Anot+IE+11%0Anot+dead%0A) for the JS syntax.

The latest changes in the codebase are using modern JS syntax where possible, so you should be able to change it here as well.

If compatibility is an issue, we can always add `NodeList.forEach` to the list of polyfills we check the support for.

Reply:
Ah, that was exactly what I was looking for (I am a bit surprised, though, that `defaults not IE 11 not dead` covers less that 80 % of users worldwide). Fixed.

I'll make a separate PR for removing this pattern from the entire codebase.